src/test/java/org/broadinstitute/consent/http/db/DarCollectionSummaryDAOTest.java

```diff
@@ -963,4 +964,76 @@ void testGetDarCollectionSummaryByCollectionId_ArchivedCollection() {
         archivedCollectionId);
     assertNull(summary);
   }
+
+  @Test
+  void testGetDarCollectionSummaryWithProgressReports() {
+    Dac dac = createDac();
+    User user = createUserWithInstitution();
+    User userChair = createUserWithRoleInDac(UserRoles.CHAIRPERSON.getRoleId(), dac.getDacId());
+    Integer userId = user.getUserId();
+    Integer chairId = userChair.getUserId();
+    Integer dacId = dac.getDacId();
+    Dataset dataset = createDatasetWithDac(userId, dacId);
+
+
+    DarCollectionSummary summary = createDarWithVotes(userId, chairId, dataset.getDatasetId());
+
+    List<DarCollectionSummary> summariesForResearcher = darCollectionSummaryDAO.getDarCollectionSummariesForResearcher(userId);
+    assertEquals(1, summariesForResearcher.size());
+    DataAccessRequest dar = dataAccessRequestDAO.findByReferenceId(summariesForResearcher.get(0).getReferenceIds().stream().findFirst().get());
+
+    assertEquals(summary.getDarCollectionId(), dar.getCollectionId());
+    //create child progress report
+    String progressReportReferenceId = createProgressReportFromDAR(dar);
+    validateSummaryObjectWithParent(userId, progressReportReferenceId, dar);
+    //create grandchild progress report
+    DataAccessRequest progressReportDar = dataAccessRequestDAO.findByReferenceId(progressReportReferenceId);
+    String progressReportReferenceId2 = createProgressReportFromDAR(progressReportDar);
+    validateSummaryObjectWithParent(userId, progressReportReferenceId2, progressReportDar);
+    //create a second grandchild, peer to the first
```

Comment:
Yes, our model currently supports this!
Closing this path will mean we can likely shift to building a LinkedList or some other order preserving structure to simplify to the "chaining" model we spoke about.  I do think we need to understand our PO's perspective on if they think we'll need to support branching before making the change away from this.

Reply:
Agreed that we need to confirm the chained approach with product or go back to the tech design phase.

Reply:
Removed this part of the test since we're still deciding if we support it or not (outside the scope of this ticket).